frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -20,9 +20,12 @@ SPDX-License-Identifier: Apache-2.0
       </v-chip>
     </template>
     <v-list class="pa-0">
-      <v-list-item v-for="({title, value, description}) in workerGroupDescriptions" :key="title" class="px-0">
+      <v-list-item v-for="({title, architecture, value, description}) in workerGroupDescriptions" :key="title" class="px-0">
         <v-list-item-content class="pt-1">
-          <v-list-item-subtitle>{{title}}</v-list-item-subtitle>
+          <v-list-item-subtitle>
+            {{title}}
+            <v-chip v-if="architecture" color="primary" label x-small class="ml-2 px-1" :key="architecture" outlined>{{architecture}}</v-chip>
```

Comment:
<img width="510" alt="Screenshot 2022-08-02 at 17 11 41" src="https://user-images.githubusercontent.com/5526658/182409583-c229617a-3cb1-421f-b1e7-67da4432815a.png">

We have the `Machine Type` title and below the actual machine type as one would expect from the title but also the CPU, GPU, Memory information in brackets and the title has has an architecture chip (without tooltip)

The `Machine Image` section shows the machine image name but also the version, separated by the pipe character.

This is not really consistent and structured. We should consider to improve this

Reply:
I added an issue for this: #1279
And added an additional Architecture section